src/Connections/FeedConnection.ts

```diff
@@ -182,6 +185,15 @@ export class FeedConnection extends BaseConnection implements IConnection {
     }
 
     public async handleFeedEntry(entry: FeedEntry): Promise<void> {
+        // We will need to tidy this up.
+        if (this.state.template?.includes('$SUMMARY') && entry.summary) {
+            // This might be massive and cause us to fail to send the message
+            // so confine to a maximum size.
+            if (entry.summary.length > MAX_SUMMARY_LENGTH) {
+                entry.summary = entry.summary?.substring(0, MAX_SUMMARY_LENGTH) + "…" ?? null;
```

Comment:
`entry.summary` is guaranteed to be non-nullish here, right?  If that's the case, then this can be:
```suggestion
                entry.summary = entry.summary.substring(0, MAX_SUMMARY_LENGTH) + "…";
```